node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
I agree it is pure convenience function. So should this PR be closed? I can add documentation for the `value` property to [this PR](https://github.com/tessel/t2-docs/pull/16) to finish it off.